substrate/frame/broker/src/adapt_price.rs

```diff
@@ -135,6 +136,44 @@ impl<Balance: FixedPointOperand> AdaptPrice<Balance> for CenterTargetPrice<Balan
 	}
 }
 
+/// `AdaptPrice` like `CenterTargetPrice`, but with a minimum price.
+///
+/// This price adapter behaves exactly like `CenterTargetPrice`, except that it takes a minimum
+/// price and makes sure that the returned `end_price` is never lower than that.
+///
+/// Target price will also get adusted if necessary (it will never be less than the end_price).
+pub struct MinimumPrice<Balance, MinPrice>(core::marker::PhantomData<(Balance, MinPrice)>);
```

Comment:
We don't really need this pr? We can just move this to the `coretime-chain` in the fellowship repo and add the minimum price check?

Reply:
Except that we do need some pallet to have the minimum price configurable. Also this PR adjusts renewals if they drop too low compared to current market price.

Reply:
Interesting, wasn't aware of this pallet. Are we talking about [this one](https://github.com/paritytech/polkadot-sdk/blob/2a233c182921f30b590c9e524f3179134494fdde/substrate/frame/parameters/src/lib.rs#L23). If so, it says it should not be used in production?